extension/options.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <style>
+    html {
```

Comment:
These default styles on `html` aren't necessary. It seems that the stylesheets introduced by [`browser_style: true`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_styles) are applied automatically. 

You can explicitly disable to see the difference:

```diff 
  "options_ui": {
    "page": "options.html",
+   "browser_style": false
  },
```

Reply:
Ah, I had forgotten this which you told me at the all hands.
Sorry and thanks!